src/main/groovy/org/labkey/gradle/task/ModuleDistribution.groovy

```diff
@@ -474,9 +470,9 @@ class ModuleDistribution extends DefaultTask
 
         ant.zip(destfile: getEmbeddedZipArchivePath()) {
             zipfileset(dir: project.buildDir, prefix: archiveName) { include(name: serverJarFile.getName()) }
-            zipfileset(dir: utilsDir.path, prefix: "${archiveName}/bin")
-            zipfileset(dir: project.buildDir, prefix: archiveName, filemode: 744){
-                include(name: "manual-upgrade.sh")
+
+            if (!simpleDistribution) {
+                zipfileset(dir: utilsDir.path, prefix: "${archiveName}/bin")
```

Comment:
Are you sure you can get rid of the setting of the filemode for `manual-upgradle.sh`?  Are we no longer building installers from a Windows machine?  I think that's where this need arose.

Reply:
This removes that script from embedded distributions altogether. I don't think it actually works for embedded distributions.

Reply:
Got it.  Didn't see the larger context here.